_data/merchants.yml

```diff
@@ -170,6 +170,8 @@
       url: https://silverround.com/
     - name: Synntech Gaming
       url: https://synntechgaming.com
+    - name: All For Crypto
```

Comment:
Could we move this up to the top of the section to retain the alphabetical listing?

Reply:
Sorry, I forgot about that. I think it would be better to sort alphabetically in the liquid template so that the order in the `merchants.yml` has no effect on the ordering on the page. I'll create a PR for that in a few minutes.

Reply:
Awesome. I still think it may be beneficial to keep it ordered as the list grows as it makes it easier to maintain. (Making sure not to duplicate items is easier.)